modules/metrics/README.md

```diff
@@ -203,6 +204,22 @@ ffwd.discovery = {
 }
 ```
 
+### duration-threshold-metrics
+
+key | type | required | note
+--- | ---- | -------- | ----
+`duration-threshold-metrics.{endpoint}.{method}` | int | required | Duration threshold in milliseconds for an endpoint/method combination
```

Comment:
I can't tell by looking at this name what it is supposed to convey, also having `metrics` in the name is redundant.

Maybe a better name would be endpoint-duration-goal/endpoint-duration-slo ?